pkg/backplaneapi/deprecation.go

```diff
@@ -0,0 +1,18 @@
+package backplaneapi
```

Comment:
This is a good package .. we should move the api-client also to this package ( as a separate task )